src/test/java/camera/CameraChestDetectorTest.java

```diff
@@ -38,7 +38,7 @@ void includeChestContoursInFrameCallTest() throws InterruptedException {
         while (ch.isChanged()) {
             ch.processFrames();
         }
-        TimeUnit.SECONDS.sleep(5);
+        TimeUnit.SECONDS.sleep(SLEEPYTIME);
```

Comment:
This is a little bit unneeded, because this will be removed in GameStatusPane branch, but ok if you want to do it

Reply:
checkstyle said it was a magicnumber and since gamestatuspane is not added yet I thought I might as well change it here and now

Reply:
yes i begrijp maar still i wanted to zeik een beetje.

Reply:
ok